charts/umbrella/values.yaml

```
@@ -1321,3 +1324,50 @@ bdrs-server-memory:
         className: "nginx"
         tls:
           enabled: false
+
+ssi-dim-wallet-stub:
+  enabled: false
+  wallet:
+    replicaCount: 1
+    host: ssi-dim-wallet-stub.tx.test
+    nameSpace: "umbrella"
+    appName: "ssi-dim-wallet-stub"
+    configName: "ssi-dim-wallet-config"
+    serviceName: "ssi-dim-wallet-service"
+    secretName: "ssi-dim-wallet-secret"
+    ingressName: "ssi-dim-wallet-ingress"
+    image:
+      repository: tractusx/ssi-dim-wallet-stub
+      pullPolicy: IfNotPresent
+      tag: "latest"
+    ingress:
+      enabled: true
+      tls: false
+      urlPrefix: /
+      className: nginx
+      annotations: {}
+    swagger:
+        ui:
+          status: true
+        apiDoc:
+          status: true
+    logLevel: "debug"
+    environment: "default"
+    baseWalletBpn: "BPNL000000000000"
+    didHost: "ssi-dim-wallet-stub.tx.test"
+    stubUrl: "http://ssi-dim-wallet-stub.tx.test"
+    statusListVcId: "8a6c7486-1e1f-4555-bdd2-1a178182651e"
+    tokenExpiryTime: "5"
+    portal:
+      waitTime: "60"
+      host: "http://portal-backend.tx.test"
+      clientId: "sa-cl2-05"
+      clientSecret: "zuBGfr67Tj0WJ5fAJSIRvoPEP5hSQFMT"
```

Should be changed to once https://github.com/eclipse-tractusx/tractus-x-umbrella/pull/127 is merged
```suggestion
        clientSecret: "changeme"
```